pegasus/sites.v3/csedweek.org/public/index.haml

```diff
@@ -17,8 +17,8 @@ social:
   "twitter:url": "https://csedweek.org/"
   "twitter:title": "<%= I18n.t(:csedweek_og_title) %>"
   "twitter:description": "<%= I18n.t(:csedweek_og_description)%>"
-  "twitter:image:src": "https://csedweek.org/images/hour-of-code-2014-video-thumbnail.jpg"
-  "twitter:player": "https://youtube.com/v/KsOIlDT145A"
+  "twitter:image:src": "https://hourofcode.com/images/hourofcode-2015-video-thumbnail.png"
```

Comment:
Ah! Yes. I just noticed it still had the 2014 image :P Which is basically the same except for the # of participants on the image.